rootfs/deploy.py

```diff
@@ -67,3 +67,6 @@ def log_output(stream, decode):
 print("pushing to registry")
 stream = client.push(registry+'/'+imageName, tag=imageTag, stream=True, insecure_registry=True)
 log_output(stream, False)
+print("deleting image")
+stream = client.remote_image(image=registry+'/'+imageName, force=True, noproune=False)
```

Comment:
haha, nope - it's supposed to be `remove_image`. fixed in https://github.com/deis/dockerbuilder/pull/39/commits/28db3ddcdba773cedad6f57f327adb7ac37f8bd7